python/lsst/ts/standardscripts/auxtel/mock/attcs_mock.py

```diff
@@ -0,0 +1,296 @@
+
+        self.atdome.cmd_moveAzimuth.callback = self.fake_move_dome
+
+    async def start_task_publish(self):
+
+        if self.start_task_finished:
+            return
+
+        await asyncio.gather(self.atmcs.start_task,
+                             self.atptg.start_task,
+                             self.atdome.start_task,
+                             self.ataos.start_task,
+                             self.atpneumatics.start_task,
+                             self.athexapod.start_task,
+                             self.atdometrajectory.start_task)
+
+        for comp in self.components:
+            getattr(self, comp).evt_summaryState.set_put(summaryState=salobj.State.STANDBY)
+            self.setting_versions[comp] = f"test_{comp}"
+            getattr(self, comp).evt_settingVersions.set_put(
+                recommendedSettingsVersion=f"{self.setting_versions[comp]},"
+            )
+
+        self.atdome.evt_scbLink.set_put(active=True, force_output=True)
+        self.run_telemetry_loop = True
+        self.tel_pos_task = asyncio.create_task(self.fake_tel_pos_telemetry())
+        self.dome_pos_task = asyncio.create_task(self.dome_tel_pos_telemetry())
+
+        self.start_task_finished = True
+
+    async def fake_tel_pos_telemetry(self):
+        while self.run_telemetry_loop:
+
+            self.atmcs.tel_mount_AzEl_Encoders.set_put(
+                elevationCalculatedAngle=np.zeros(100)+self.tel_alt,
+                azimuthCalculatedAngle=np.zeros(100)+self.tel_az,
+            )
+
+            if self.track:
+                self.atmcs.evt_target.set_put(elevation=self.tel_alt,
+                                              azimuth=self.tel_az,
+                                              force_output=True)
+
+            await asyncio.sleep(1.)
+
+    async def dome_tel_pos_telemetry(self):
+        while self.run_telemetry_loop:
+            self.atdome.tel_position.set_put(azimuthPosition=self.dom_az)
+            await asyncio.sleep(1.)
+
+    async def atmcs_wait_and_fault(self, wait_time):
+        self.atmcs.evt_summaryState.set_put(summaryState=salobj.State.ENABLED,
+                                            force_output=True)
+        await asyncio.sleep(wait_time)
+        self.atmcs.evt_summaryState.set_put(summaryState=salobj.State.FAULT,
+                                            force_output=True)
+
+    async def atptg_wait_and_fault(self, wait_time):
+        self.atptg.evt_summaryState.set_put(summaryState=salobj.State.ENABLED,
+                                            force_output=True)
+        await asyncio.sleep(wait_time)
+        self.atptg.evt_summaryState.set_put(summaryState=salobj.State.FAULT,
+                                            force_output=True)
+
+    async def fake_slew_callback(self, id_data):
+        """Fake slew waits 5 seconds, then reports all axes
+           in position. Does not simulate the actual slew.
```

Comment:
Picky but..what does "Does not simulate the actual slew" mean? I suspect you mean that it only outputs inPosition events, in which case perhaps just say that?

Again I don't think these "fake_" prefixes add useful information. Everything is fake in this Mock; it doesn't delegate anything to the real CSCs.